Meter.c

```diff
@@ -345,8 +344,11 @@ static void GraphMeterMode_draw(Meter* this, int x, int y, int w) {
    }
    for (; i < nValues - 1; i += 2, k++) {
       int pix = GraphMeterMode_pixPerRow * GRAPH_HEIGHT;
-      int v1 = CLAMP((int) lround(data->values[i] * pix), 1, pix);
-      int v2 = CLAMP((int) lround(data->values[i + 1] * pix), 1, pix);
+      if (this->total == 0) {
```

Comment:
```suggestion
      if (this->total < 1) {
```
Rationale:
* (nearly) never compare float values to zero
* we had the issue with very small denominators scaling the iometers very high before

Reply:
But why would we limit the code to total being bigger than 1? I totally see use cases where we might want to display something that moves on the scale of 0 to 0.1, when clamping it to 1 the graph will become useless in such cases?

Reply:
Well, <ins>you</ins> made the zero case jump to one:
```C
if (this->total < FLT_EPSILON)
   this->total = 1;
```

I'd be fine with 1 as currently there is no meter that has a use for a total < 1.
But you can clamp to "an arbitrary small constant like 0.0001" as you suggest below as well. Doesn't matter.
Just not use a C constant wrongly and please make the function continuous, so if `total < ε` then also clamp total to ε.